api/graphql/schema.graphql

```diff
@@ -108,7 +108,7 @@ type User {
     # Profile Status
     isProfileComplete: Boolean
     expectedGenericJobTitles: [GenericJobTitle] @belongsToMany
-
+    priorityDerived: PriorityDerivedStatus
```

Comment:
I think that the "Dervied" suffix is an implementation detail and shouldn't be leaked through the API.  I'd prefer something like what Eric suggested - priorityWeight.